src/hazelcore/dynamics/DHExp.re

```diff
@@ -141,6 +141,7 @@ type t =
   // TODO rename to ExpandingKeyword
   | Keyword(MetaVar.t, MetaVarInst.t, VarMap.t_(t), ExpandingKeyword.t)
   | FreeVar(MetaVar.t, MetaVarInst.t, VarMap.t_(t), Var.t)
+  | Duplicate(MetaVar.t, MetaVarInst.t, VarMap.t_(t), Var.t)
```

Comment:
Why does `DHExp` have a duplicate variant? Doesn't that only apply to pattern variables?